src/python/test/test_dtm_rips_complex.py

```diff
@@ -30,3 +34,9 @@ def test_compatibility_with_rips():
     st_from_rips = rips_complex.create_simplex_tree(max_dimension=1)
     assert list(st.get_filtration()) == list(st_from_rips.get_filtration())
 
+
+def test_empty_dtm_rips_complex():
+    dtm_rips = DTMRipsComplex()
+    st = dtm_rips.create_simplex_tree(max_dimension=1)
+    assert st.num_simplices() == 1
```

Comment:
Why is there a vertex if there are no points?

Reply:
Was part of your other comment. Fixed on [98e1ce1](https://github.com/GUDHI/gudhi-devel/pull/999/commits/98e1ce14bd2ddb3475d06f88b3c61c3b846afabc)